CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Faro
 
+## Version 1.0.0
+
+### **WARNING**
+Version 1.0.0 is not compatible with 0.7.2. All the functions, classes, protocols, ... that we no longer support have been marked with a ```@available(*, deprecated=1.0.0, message="use the Bar class")``` attribute.
+
+In the previous versions to fire a service request you used the ```Air``` class. Now you should use the ```Bar``` class.
```

Comment:
🎈The sentence is a bit weird.

In **AirRivet** you had to use the `Air` class to trigger a service request, this has been replaced by the `Bar` class in **Faro**.